src/MoBi.Presentation/Tasks/SimulationUpdateTask.cs

```diff
@@ -77,8 +77,9 @@ public ICommand ConfigureSimulationAndAddToProject(IMoBiSimulation clonedSimulat
             CommandType = AppConstants.Commands.AddCommand,
             Description = AppConstants.Commands.AddToProjectDescription(ObjectTypes.Simulation, clonedSimulation.Name)
          };
-
-         macroCommand.Add(ConfigureSimulation(clonedSimulation));
+         // I Removed this since the simulation will be removed on reversal, and then try to configure it. 
+         // Therefore, it will fail since after removal, the configuration will try to be performed over a null simulation.
+         ConfigureSimulation(clonedSimulation);
```

Comment:
Was the comment supposed to be a PR comment? somehow it does not belong into the code